sim/pokemon.js

```diff
@@ -611,18 +613,18 @@ class Pokemon {
 	gotAttacked(move, damage, source) {
 		if (!damage) damage = 0;
 		move = this.battle.getMove(move);
-		let lastHurtBy = {
+		let lastAttackedBy = {
 			source: source,
 			damage: damage,
 			move: move.id,
 			thisTurn: true,
 		};
-		this.hurtBy.push(lastHurtBy);
+		this.attackedBy.push(lastAttackedBy);
 	}
 
-	getLastHurtBy() {
-		if (this.hurtBy.length === 0) return undefined;
-		return this.hurtBy[this.hurtBy.length - 1];
+	getLastAttackedBy() {
+		if (this.attackedBy.length === 0) return undefined;
+		return this.attackedBy[this.attackedBy.length - 1];
```

Comment:
It looks like you made a mistake while rebasing. Can you fix this?

Reply:
Oh, this was an intentional rename. Nevermind then.

Reply:
Are you referring to my renaming of `hurtBy` to `attackedBy`? I just felt that `hurtBy` and `hurtThisTurn` were confusing, since they are tracking different types of damage. I can change it back if you prefer.

Reply:
Ok, cool.